src/main/java/io/antmedia/filter/AbstractFilter.java

```diff
@@ -32,6 +38,36 @@ public AppSettings getAppSettings()
 		return appSettings;
 	}
 	
+	public ServerSettings getServerSetting() 
+	{
+		ServerSettings serverSettings = null;
+		ConfigurableWebApplicationContext context = getAppContext();
+		if (context != null) {
+			serverSettings = (ServerSettings)context.getBean(ServerSettings.BEAN_NAME);
+		}
+		return serverSettings;
+	}
+	
+	public boolean checkCIDRList(List<NetMask> allowedCIDRList, final String property) {
```

Comment:
Please make the parameter name meaningful. What kind of property it is?

Meaningful names makes easy understand the code